modules/webhook-configuring-acs.adoc

```diff
@@ -10,19 +10,19 @@ Create a new integration in {product-title} by using the webhook URL.
 
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
-. Click *New Integration* (*`add`* icon).
-. Enter a name for *Integration Name*.
+. Scroll down to the *Notifier Integrations* section and select *Generic Webhook*.
+. Click *New integration*.
+. Enter a name for *Integration name*.
 . Enter the webhook URL in the *Endpoint* field.
-. If your webhook receiver uses an untrusted certificate, enter a CA certificate in the *CA Cert* field; otherwise leave it blank.
+. If your webhook receiver uses an untrusted certificate, enter a CA certificate in the *CA certificate* field; otherwise leave it blank.
 +
 [NOTE]
 ====
 The server certificate used by the webhook receiver must be valid for the endpoint DNS name.
-You can turn on the *Skip TLS Verify* toggle to ignore this validation.
+You can click *Skip TLS verification* to ignore this validation.
 Red Hat does not recommend turning off TLS verification.
```

Comment:
Hi Kerry! I know this is not explicitly in this PR, but if you have the time the [IBM Word usage guides](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r) advises to change:
`recommend` -> `suggest`